pytorch_segmentation_detection/recipes/pascal_voc/segmentation/resnet_34_8s_train.ipynb

```diff
@@ -11,8 +11,8 @@
     "%matplotlib inline\n",
     "\n",
     "import sys, os\n",
-    "sys.path.append(\"/home/daniil/repos/pytorch-segmentation-detection/\")\n",
-    "sys.path.insert(0, '/home/daniil/repos/pytorch-segmentation-detection/vision/')\n",
+    "sys.path.insert(0, '../../../../vision/')\n",
```

Comment:
Would be great if you can just define a variable with path to the cloned github repository
which will be used in ```sys.path()```